### PR TITLE
fix(kit): correctly loose box shadow on tile after dragged

### LINKED ISSUE #7157: 🐞 - Still active element in tiles
### Playground Link

https://taiga-ui.dev/next/components/tiles

### Description

Try reorder

image

### Angular version

latest

### Taiga UI version

latest

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
Fixes #7157

### CODE REVIEW

**Comment by splincode** [comment]
01-tiles-drag-and-drop-chromium-darwin
02-tiles-drag-and-drop-chromium-darwin

File: projects/demo-playwright/tests/kit/tiles/tiles.pw.spec.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+test.describe('Tiles', () => {
+    test('correctly loose box shadow after dragged', async ({page}) => {
+        await tuiGoto(page, '/components/tiles#vertical');
+
+        const example = new TuiDocumentationPagePO(page).getExample('#vertical');
+        const first = example.locator('tui-tile').first();
+        const last = example.locator('tui-tile').last();
+
+        await expect(example).toHaveScreenshot('01-tiles-drag-and-drop.png');
```
Comment (splincode) [inline projects/demo-playwright/tests/kit/tiles/tiles.pw.spec.ts:12]: 01-tiles-drag-and-drop-chromium-darwin

File: projects/demo-playwright/tests/kit/tiles/tiles.pw.spec.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+        await first.dragTo(last);
+        await page.waitForTimeout(300);
+
+        await expect(example).toHaveScreenshot('02-tiles-drag-and-drop.png');
```
Comment (splincode) [inline projects/demo-playwright/tests/kit/tiles/tiles.pw.spec.ts:17]: 02-tiles-drag-and-drop-chromium-darwin

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo-playwright/tests/kit/tiles/tiles.pw.spec.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe('Tiles', () => {
+    test('correctly loose box shadow after dragged', async ({page}) => {
+        await tuiGoto(page, '/components/tiles#vertical');
```
Comment (mdlufy) [inline projects/demo-playwright/tests/kit/tiles/tiles.pw.spec.ts:6]: `DemoRoute.Tiles`

Comment (splincode) [reply projects/demo-playwright/tests/kit/tiles/tiles.pw.spec.ts]: fixed

**Pull request merged**